tests/openvas_agent_test.py

```diff
@@ -83,6 +83,7 @@ def testAgentOpenVas_whenBinaryAvailable_RunScan(
             risk_rating=vuln_utils.RiskRating.INFO,
             technical_detail=f"\n```json\n{json.dumps(output, indent=4, sort_keys=True)}\n```",
             vulnerability_location=vulnerability_location,
+            dna='{"location": {"ipv4": {"host": "128.0.0.1", "mask": "32", "version": 4}, "metadata": [{"type": "PORT", "value": "80"}]}, "title": ""}',
```

Comment:
Title is empty in this case?

Reply:
There was a bug in the code, when the title is None we default to a title but the title was an empty string in that case!!